<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrame</class>
 <widget class="QFrame" name="MainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#MainFrame {background-image: url(:/Images/resources/images/KPA_MainWindow_Dark_Skin.png);}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>1</number>
  </property>
  <widget class="QToasterDial" name="qGainDial">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="qVolumeDial">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>480</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="minValue">
    <double>-96.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>12.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QToasterDial" name="qNoiseGateDial">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="qMasterVolumeDial">
   <property name="geometry">
    <rect>
     <x>785</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Big</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="qModRateDial">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
  </widget>
  <widget class="QToasterDial" name="qModIntensityDial">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
  </widget>
  <widget class="QToasterDial" name="qDelayFeedbackDial">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>200.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string>%</string>
   </property>
  </widget>
  <widget class="QToasterDial" name="qDelayMixDial">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>200.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string>%</string>
   </property>
  </widget>
  <widget class="QToasterDial" name="qReverbTimeDial">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="qReverbMixDial">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>200.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string>%</string>
   </property>
  </widget>
  <widget class="QToasterDial" name="qCleanSenseDial">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>18</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="minValue">
    <double>-12.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>12.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QToasterDial" name="qDistortionSenseDial">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>84</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="minValue">
    <double>-12.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>12.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QToasterDial" name="qHeadphoneVolumeDial">
   <property name="geometry">
    <rect>
     <x>803</x>
     <y>18</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="minValue">
    <double>-96.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>0.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>1.000000000000000</double>
   </property>
   <property name="unit">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QToasterDial" name="qMonitorVolumeDial">
   <property name="geometry">
    <rect>
     <x>761</x>
     <y>84</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="minValue">
    <double>-96.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>0.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QToasterButton" name="qStompAButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>74</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="qEQButton">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>64</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="qStompBButton">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>64</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="qStompCButton">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>64</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="qStompDButton">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>74</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="qStompXButton">
   <property name="geometry">
    <rect>
     <x>576</x>
     <y>74</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="qStompModButton">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>64</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="qRigUpButton">
   <property name="geometry">
    <rect>
     <x>806</x>
     <y>331</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="state">
    <enum>QToasterButton::Off</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="qRigNextButton">
   <property name="geometry">
    <rect>
     <x>848</x>
     <y>369</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="state">
    <enum>QToasterButton::On</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="qRigDownButton">
   <property name="geometry">
    <rect>
     <x>806</x>
     <y>406</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="state">
    <enum>QToasterButton::Off</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="qRigPrevButton">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>369</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="state">
    <enum>QToasterButton::On</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="qAmplifierButton">
   <property name="geometry">
    <rect>
     <x>379</x>
     <y>74</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="qCabinetButton">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>74</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="qStompDelayButton">
   <property name="geometry">
    <rect>
     <x>663</x>
     <y>64</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="qStompReverbButton">
   <property name="geometry">
    <rect>
     <x>716</x>
     <y>74</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="qStompsButton">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>118</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="qStackButton">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>118</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="qEffectsButton">
   <property name="geometry">
    <rect>
     <x>633</x>
     <y>118</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QChickenHeadDial" name="qChickenHeadDial">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>346</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="smallDialStackedWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>460</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget#smallDialStackedWidget {color: transparent;}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="browserPage">
    <property name="styleSheet">
     <string notr="true">QWidget#browserPage {
background-color: transparent;}</string>
    </property>
    <widget class="QToasterDial" name="qToasterDial_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
    </widget>
    <widget class="QToasterDial" name="qToasterDial_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
    </widget>
    <widget class="QToasterDial" name="qToasterDial_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
    </widget>
    <widget class="QToasterEnumDial" name="qToasterEnumDial">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterEnumDial::Small</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="stompEditorPage">
    <widget class="QStackedWidget" name="stompStackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>460</width>
       <height>80</height>
      </rect>
     </property>
     <widget class="QWidget" name="wahWahPage"/>
     <widget class="QWidget" name="waLowPassPage"/>
    </widget>
   </widget>
  </widget>
  <widget class="QToasterLCD" name="qLCDDisplay">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>335</y>
     <width>420</width>
     <height>115</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="color">
    <enum>QToasterLCD::Standard</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="bigDialsStackedWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>460</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget#bigDialsStackedWidget{
	background-color: transparent;
}</string>
   </property>
   <widget class="QWidget" name="eqPage">
    <property name="styleSheet">
     <string notr="true">QWidget#eqPage{
	background-color: transparent;
}</string>
    </property>
    <widget class="QToasterDial" name="qEqBassDial">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-5.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>5.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QToasterDial" name="qEqMiddleDial">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-5.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>5.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QToasterDial" name="qEqTrebleDial">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-5.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>5.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QToasterDial" name="qEqPresenceDial">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-5.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>5.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QToasterDial</class>
   <extends>QWidget</extends>
   <header>qtoasterdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterEnumDial</class>
   <extends>QWidget</extends>
   <header>qtoasterenumdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterLCD</class>
   <extends>QStackedWidget</extends>
   <header>QToasterLCD.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QToasterButton</class>
   <extends>QWidget</extends>
   <header>qtoasterbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QChickenHeadDial</class>
   <extends>QWidget</extends>
   <header>qchickenheaddial.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qToasterEnumDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>qLCDDisplay</receiver>
   <slot>setBrowserModeView(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qLCDDisplay</sender>
   <signal>browserModeViewChanged(int)</signal>
   <receiver>qToasterEnumDial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
